eventstore/mongodb_v2/eventstore.go

```diff
@@ -33,11 +34,15 @@ import (
 	"github.com/looplab/eventhorizon/uuid"
 )
 
+// ErrCollectionNamesEqual is when provided custom collection names are equal.
+var ErrCollectionNamesEqual = errors.New("custom collection names are equal")
```

Comment:
You don't need a custom error for this. I usually go by the guideline to only add such if someone need them to react on and do something else. In this case an app would probably only quit if this error occurs, not retry.